string prop_label
float64 radius
float64 angle